wp-content/themes/pub/wporg-learn-2020/css/components/_workshop-discussion-events.scss

```diff
@@ -44,5 +41,10 @@
 	.wp-block-button__link:visited {
 		margin: 0;
 		padding: 0.5rem;
+		white-space: nowrap;
```

Comment:
Keep the buttons a consistent width, so that they don't get squashed due to the flex layout